pkg/eks/fargate_test.go

```diff
@@ -73,4 +73,11 @@ var _ = Describe("Fargate", func() {
 			})
 		})
 	})
+
+	Describe("DoListFargateProfiles", func() {
+		It("should list profiles", func() {
+			fakeClient.ListProfilesReturns([]string{fargateProfileName}, nil)
+			Expect(eks.DoListFargateProfiles(context.Background(), config, fakeClient)).To(Succeed())
+		})
+	})
```

Comment:
Not sure what to test, so I only create a simple test for ListProfiles.

Reply:
When I mentioned adding tests I meant testing to make sure the 409 error is not thrown with your changes. Sorry if I wasn't clear earlier.